ynr/apps/candidates/static/js/person_form.js

```diff
@@ -164,6 +165,12 @@ var populate_party_selects = function() {
   allPartySelects.attr("disabled", false);
   $(PARTY_WIDGET_INPUT_CLASS).hide();
   allPartySelects.each(setup_single_party_select);
+
+  $(PREVIOUS_PARTY_AFFILIATIONS_SELECT_CLASS).select2({
```

Comment:
I've not tested this locally, but does this allow multiple selections?

Reply:
Yes - can test on the bulk add form, it initialises all of the fields
![Screenshot 2022-03-17 at 09 49 01](https://user-images.githubusercontent.com/15347726/158782914-e18ad316-b429-47f0-aacc-6e2bd8984c67.png)